node/src/types/block.rs

```diff
@@ -859,7 +895,7 @@ impl Display for BlockSignatures {
 pub struct Block {
     hash: BlockHash,
     header: BlockHeader,
-    body: (), // TODO: implement body of block
+    body: BlockBody,
```

Comment:
Might be easier to just put the three fields directly into `Block`; I imagine we won't be using the `BlockBody` without the header anywhere?
(Just a thought; I don't have a strong opinion on that.)